compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/Desugar.java

```diff
@@ -794,6 +794,8 @@ public void visit(BLangPackage pkgNode) {
 
         pkgNode.constants = removeDuplicateConstants(pkgNode);
 
+        rewrite(pkgNode.typeDefinitions, env);
```

Comment:
Why aren't we assigning the value to `pkgNode.typeDefinitions` here as previous?

Reply:
Earlier also, we didn't need to do that because within the rewrite function, we set the desugared value to the node list.